stacks/python/devfile.yaml

```diff
@@ -16,14 +16,14 @@ starterProjects:
 components:
   - name: py-web
     container:
-      image: quay.io/eclipse/che-python-3.7:nightly
+      image: ubi9/python-39:latest
```

Comment:
Likewise, this image doesn't exist either. You need to specify the full registry server in the image: `registry.access.redhat.com/ubi9/python-39:latest`